examples/01_bars.html

```diff
@@ -48,21 +48,21 @@ <h2>01 _ Bars</h2>
 
 					var elem = { x: startValue, domElement: domElement };
 
-					var updateCallback = function() {
-						this.domElement.style.left = this.x + 'px';
+					var updateCallback = function(elem) {
+						elem.domElement.style.left = elem.x + 'px';
```

Comment:
Don't think you need the `elem` arg here, and also no `.bind` because the `var elem` is in scope, that'd be cleaner.